packages/datx-codemod/src/bin/cli.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+// Based on https://github.com/reactjs/react-codemod/blob/dd8671c9a470a2c342b221ec903c574cf31e9f57/bin/cli.js
+// @datx/codemod optional-name-of-transform optional/path/to/src [...options]
+
+import globby from 'globby';
+import inquirer from 'inquirer';
+import meow from 'meow';
+import type { Options, AnyFlags } from 'meow';
+import path from 'path';
+import execa from 'execa';
+import chalk from 'chalk';
+import isGitClean from 'is-git-clean';
+
+export const jscodeshiftExecutable = require.resolve('.bin/jscodeshift');
+export const transformerDirectory = path.join(__dirname, '../', 'transforms');
+
+export function checkGitStatus(force) {
+  let clean = false;
+  let errorMessage = 'Unable to determine if git directory is clean';
+  try {
+    clean = isGitClean.sync(process.cwd());
+    errorMessage = 'Git directory is not clean';
+  } catch (err) {
+    if (err && err.stderr && err.stderr.indexOf('Not a git repository') >= 0) {
+      clean = true;
+    }
+  }
+
+  if (!clean) {
+    if (force) {
+      console.log(`WARNING: ${errorMessage}. Forcibly continuing.`);
```

Comment:
```suggestion
      console.warn(`WARNING: ${errorMessage}. Forcibly continuing.`);
```

Reply:
Does it matter? 
it's practically the same source code as the facebooks cli https://github.com/reactjs/react-codemod/blob/dd8671c9a470a2c342b221ec903c574cf31e9f57/bin/cli.js#L36 and they use warn

Reply:
Leave as is then 😄